gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -288,13 +287,28 @@ public void orchestrate(Spec spec) throws Exception {
       flowMetadata.putIfAbsent(TimingEvent.FlowEventConstants.FLOW_EXECUTION_ID_FIELD,
           jobExecutionPlanDag.getNodes().get(0).getValue().getJobSpec().getConfigAsProperties().getProperty(ConfigurationKeys.FLOW_EXECUTION_ID_KEY));
 
-      if (flowCompilationTimer != null) {
-        flowCompilationTimer.stop(flowMetadata);
+      if (flowCompilationTimer.isPresent()) {
+        flowCompilationTimer.get().stop(flowMetadata);
       }
 
       if (this.dagManager.isPresent()) {
-        //Send the dag to the DagManager.
-        this.dagManager.get().addDag(jobExecutionPlanDag, true, true);
+        try {
+          //Send the dag to the DagManager.
+          this.dagManager.get().addDag(jobExecutionPlanDag, true, true);
+        } catch (Exception ex) {
```

Comment:
How was this error handled previously? I would expect that it was logged somewhere. I wonder if we'll get two errors in logs for the same problem.

Reply:
`GoblinServiceJobScheduler.runJob` is the only caller, and it both catches and logs `Exception`.  I sought to avoid a duplicate stack trace (since we continue to unwinding).  I dropped the log line in, to report the flow execution ID.

would you suggest not to log here at all?  as an example, the two log entries look like:
```
2021/08/10 00:08:45.902 WARN [GobblinServiceManager] [JobScheduler-1] [shared-gobblin-service-war] [] [flow: {flowGroup=testKip, flowName=test001}; exec: 1628579325837] Failed to add 
Job Execution Plan due to: No, I won't add the DAG, dawg!
2021/08/10 00:08:45.903 ERROR [GobblinServiceManager] [JobScheduler-1] [shared-gobblin-service-war] [] Failed to run job gobblin-flow:/testKip/test001
org.apache.gobblin.runtime.JobException: Failed to run Spec: gobblin-flow:/testKip/test001
        at org.apache.gobblin.service.modules.scheduler.GobblinServiceJobScheduler.runJob(GobblinServiceJobScheduler.java:279) ~[gobblin-service-fat-1.0.46-all.jar:?]
        at org.apache.gobblin.service.modules.scheduler.GobblinServiceJobScheduler$NonScheduledJobRunner.run(GobblinServiceJobScheduler.java:476) [gobblin-service-fat-1.0.46-all.jar:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_172]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_172]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_172]
Caused by: java.io.IOException: No, I won't add the DAG, dawg!
        at org.apache.gobblin.service.modules.orchestration.DagManager.addDag(DagManager.java:266) ~[gobblin-service-fat-1.0.46-all.jar:?]
        at org.apache.gobblin.service.modules.orchestration.Orchestrator.orchestrate(Orchestrator.java:297) ~[gobblin-service-fat-1.0.46-all.jar:?]
        at org.apache.gobblin.service.modules.scheduler.GobblinServiceJobScheduler.runJob(GobblinServiceJobScheduler.java:277) ~[gobblin-service-fat-1.0.46-all.jar:?]
        ... 4 more
```

Reply:
Yeah, if it's already logged, we don't need to log it again. We have a dashboard with statistics based on number of warnings and errors in logs. It will be more accurate if each error is captured only once there.

Reply:
ok, I will remove that line, then update this PR